src/admin/webpack.config.babel.js

```diff
@@ -14,6 +14,7 @@ export default {
     },
     entry: {
         index: [
+            'babel-polyfill',
```

Comment:
You should move babel-polyfill directly to main.js in order to take full advantage of babel-preset-env. Good job @djhi !